pkg/apps/utils.jsx

```diff
@@ -60,12 +60,12 @@ export const ProgressBar = ({ size, title, data }) => {
             </SplitItem>
         </Split>);
     } else {
-        return <Progress value={data.percentage} size={size} title={title} />;
```

Comment:
I am not sure if you 're famiilar with the term of accesibility. It's essentially patterns that we use so that people with eyesight problems can still use our product. 

The problem with removing the title outside of the component is that you make it not accesible any more.

In order to make it accessible again while keeping the title outside is to use the 'aria-labelledby' as suggested in the documentation here https://www.patternfly.org/v4/components/progress

The aria-labeledby will have value the 'id' of the new 'title' element. So add a new property to the ProgressBar component here, called ariaLabeledBy where you pass the 'id' of the title element (create an id if that does not exist)